### DISCUSSION TITLE
Question about KeyBindings

### DISCUSSION THREAD

**Discussion by w2ca** [filamentphp/filament#14728] [General]
I added a "c" KeyBinding, so when I press the letter c, the CreateAction is triggered.

However, I had a problem. When I started typing, in some input, the letter c did not work.

For example, I have Task Resource. When I was going to create a new task, the action displayed a modal with the inputs. When I was going to type a word with the letter c, it did not work. I believe it triggered the action to display the modal again.

This is a bug, right?

@danharrin

**Comment by danharrin** [comment]
Yeah sounds like a bug!
